catalog/dags/common/requester.py

```diff
@@ -148,36 +137,70 @@ def _get_json(self, response) -> dict | list | None:
         except JSONDecodeError as e:
             logger.warning(f"Could not get response_json.\n{e}")
 
+    def _attempt_retry_get_response_json(
+        self,
+        error,
+        endpoint,
+        retries=0,
+        query_params=None,
+        requestMethod="get",
```

Comment:
Is there a reason for `requestMethod` being in camelCase?

Reply:
No idea, it was in camelCase already in `get_response_json` so was copied out to the new function -- but it *really* bothers me too 😅 I'll just fix it in both places!